ci/docker_cache.py

```diff
@@ -217,7 +249,11 @@ def script_name() -> str:
     args = parser.parse_args()
 
     platforms = build_util.get_platforms()
-    return build_save_containers(platforms=platforms, registry=args.docker_registry, load_cache=True)
+    try:
```

Comment:
Can we use with idiom? Python context? you can make it a separate PR

Reply:
Good idea, will do.